lib/clients/rtm/client.js

```diff
@@ -433,21 +414,21 @@ RTMClient.prototype.handleWsMessage = function handleWsMessage(wsMsg) {
   try {
     message = JSON.parse(wsMsg);
   } catch (err) {
-    // TODO(leah): Emit an event here?
-    this.logger('debug', 'unable to parse message: ' + err);
+    this.logger('error', 'Unable to parse message: ' + err);
     return;
   }
 
   if (includes(RTM_CLIENT_INTERNAL_EVENT_TYPES, message.type)) {
     this._handleWsMessageInternal(message.type, message);
   } else {
+    this._maybeKeepAlive(message);
```

Comment:
@CharlieHess help me understand this, please. `RTM_CLIENT_INTERNAL_EVENT_TYPES` includes `'pong'`, so pong messages will *not* keep the socket connection alive?